float32 pitch
float32 roll
float32 yaw
float32 pitch_rate
float32 roll_rate
float32 yaw_rate
float32 elevation
float32 altitude
float32 northings
float32 eastings
float32 latitude
float32 longitude
float32 motor_1_force
float32 motor_2_force
float32 motor_3_force
float32 motor_4_force
float32 X_Force
float32 Y_Force
float32 Z_Force
float32 K_Moment
float32 M_Moment
float32 N_Moment
float32 u_dot
float32 v_dot
float32 w_dot
float32 p_dot
float32 q_dot
float32 r_dot
float32 u
float32 v
float32 w
float32 p
float32 q
float32 r
float32 sim_time